src/WorkOS.net/Services/UserManagement/UserManagementService.cs

```diff
@@ -140,6 +140,25 @@ public UserManagementService(WorkOSClient client)
             return await this.Client.MakeAPIRequest<(User, Session)>(request, cancellationToken);
         }
 
+        /// <summary>
+        /// Creates a one-time Magic Auth code and emails it to the user.
+        /// </summary>
+        /// <param name="options"> Parameters used to send magic auth code.</param>
+        /// <param name="cancellationToken">An optional token to cancel the request.</param>
+        /// <returns> A Magic Auth Challenge.</returns>
+        public async Task<MagicAuthChallenge> SendMagicAuthCode(
```

Comment:
Looks like this endpoint returns a `{ user: { } }` response now instead of a specific challenge object.